src/Aspire.Dashboard/Model/ResourceViewModel.cs

```diff
@@ -46,28 +46,56 @@ public static string GetResourceName(ResourceViewModel resource, IEnumerable<Res
     }
 }
 
-public sealed class EnvironmentVariableViewModel(string name, string? value, bool fromSpec)
+public sealed class EnvironmentVariableViewModel
 {
-    public string Name { get; } = name;
-    public string? Value { get; } = value;
-    public bool FromSpec { get; } = fromSpec;
+    public string Name { get; }
+    public string? Value { get; }
+    public bool FromSpec { get; }
```

Comment:
```suggestion
    public string Name { get; init; }
    public string? Value { get; init; }
    public bool FromSpec { get; init; }
```

Reply:
If we make these `init` properties then the validation could be bypassed.